Dockerfile

```diff
@@ -51,3 +51,26 @@ ENV NODE_EXTRA_CA_CERTS=/usr/local/share/ca-certificates/extra-ca-certificates.c
 EXPOSE 7310
 
 CMD [ "pnpm", "start" ]
+
+########################################
+# STEP 3: EVENT-LOG-INDEXER
+########################################
+
+FROM node:18.19-bullseye-slim AS event-log-indexer
```

Comment:
Ponder uses `ast-grep` under the hood. When using the alpine image the app would crash at runtime because it couldn't find the expected architecture. For example https://github.com/ast-grep/ast-grep/tree/main/npm/platforms it might be just a bug with that version in Ponder or ast needs an update. Will look into this deeper at a later date.